tests/ui/consts/const-refcell.rs

```diff
@@ -0,0 +1,39 @@
+//@ run-pass
+//! This tests `RefCell` is usable in `const` contexts
+
+#![feature(const_ref_cell)]
+
+use std::mem;
+use std::sync::atomic::{AtomicU8, Ordering};
+use std::cell::RefCell;
+
+static DROP_COUNT: AtomicU8 = AtomicU8::new(0);
+
+struct Dummy;
+
+impl Drop for Dummy {
+    fn drop(&mut self){
+        DROP_COUNT.fetch_add(1, Ordering::Relaxed);
+    }
+}
+
+const REF_CELL_TEST: RefCell<Dummy> = {
```

Comment:
Please also try to actually change the value in the RefCell, and then observe the change in a following `borrow()`.

Reply:
Done.